lib/accumulate_into_master.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'config'
+
+_200s_directory = ARGV[0]
```

Comment:
Underscores are mostly used to signify unused variables - this is enforced by rubocop (don't know if it's in our own styleguide) https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml#L1011